GVFS/GVFS.Upgrader/GVFS.Upgrader.csproj

```diff
@@ -26,6 +26,7 @@
     <When Condition="'$(OS)' == 'Windows_NT'">
       <PropertyGroup>
         <TargetFrameworks>net461;netcoreapp2.1</TargetFrameworks>
+	<DefineConstants>$(DefineConstants);WINDOWS_BUILD</DefineConstants>

```

Comment:
Looks like this project is already referencing the Windows or the Mac Platform projects so why can't we have the platform specific code live in those projects instead of adding another way to create platform specific code?

Reply:
That is an option - but then we are putting upgrader specific code in the platform project...

Reply:
I will look at moving this into the platform projects

Reply:
I will look at moving this into the platform projects

Reply:
I will look at moving this into platform specific projects

Reply:
After thinking about this a bit more - I moved the platform specific code out of the Common project, but I am still am not sure about the `GVFS.Upgrader` project. This project is almost a separate application, and I didn't want to bring high level "other application" specific logic into the Platform classes.

I am open to thoughts on this, but I wasn't going to convert this just yet.